change/@fluentui-react-tabster-b2d43df9-1387-475e-b4e3-a4204207aba2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26584&commitId=faf9c56a43ac909cb97ad452faa444ecf2f47f04&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1828 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoButton.default - Dark Mode.active.chromium.png|626| Changed|
|InfoButton.default - High Contrast.active.chromium.png|1141| Changed|
|InfoButton.default.active.chromium.png|1126| Changed|
|InfoButton.size.rest.chromium.png|1195| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>